packages/back-end/src/scim/users/getUser.ts

```diff
@@ -0,0 +1,71 @@
+import { Response } from "express";
+import { ScimGetRequest } from "../../../types/scim";
+import { getUserById } from "../../services/users";
+
+export async function getUser(req: ScimGetRequest, res: Response) {
+  const userId = req.params.id;
+
+  const user = await getUserById(userId);
```

Comment:
We should first make sure the userId is part of the org (in the `members` array).  If not, we should pretend the user doesn't exist in our database at all.  The approach here is very risky and will likely lead to leaking information about users.